src/elements/core/datetime/date-adapter.ts

```diff
@@ -192,6 +199,15 @@ export abstract class DateAdapter<T = any> {
     return `${pad(this.getYear(date), 4)}-${pad(this.getMonth(date))}-${pad(this.getDate(date))}`;
   }
 
+  /**
+   * Retruns a date from ISO String.
+   * @param date The ISO String date to convert to T.
+   */
+  public fromIso8601(date: string): T {
```

Comment:
I think this method is already covered by `deserialize()` and should be removed here.